docs/tox.ini

```diff
@@ -0,0 +1,19 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+[tox]
+work_dir = {toxinidir}/../.tox
+requires =
+    tox>=4
```

Comment:
Could we add a line with `envlist = docs` here so that just typing `tox` builds the docs?